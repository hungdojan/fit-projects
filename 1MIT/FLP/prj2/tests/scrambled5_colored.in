ywb
gwo
gwo
orw bbr yog oww
ygo yrw bbr gob
ggo ygw bbr gyy
rob
ryy
wrr
