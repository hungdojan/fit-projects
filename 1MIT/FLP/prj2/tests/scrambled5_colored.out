ywb
gwo
gwo
orw bbr yog oww
ygo yrw bbr gob
ggo ygw bbr gyy
rob
ryy
wrr

rwb
rwo
gwo
yrw bbr yow ggo
ggo yrw bbr yow
ggo ygw bbr ybw
oob
yyy
grr

boo
www
rrg
ggo yrw bbr yow
ggo yrw bbr yow
ggo ygw bbr ybw
oob
yyy
grr

www
www
rrg
ggo yrr bbb oow
ggo yrr bbb oow
ggo ygg rrr bbw
oob
yyy
yyy

www
www
www
ggg rrr bbb ooo
ggg rrr bbb ooo
ooo ggg rrr bbb
yyy
yyy
yyy

www
www
www
ggg rrr bbb ooo
ggg rrr bbb ooo
ggg rrr bbb ooo
yyy
yyy
yyy
